content/docs/user-guide/build/troubleshooting.md

```diff
@@ -104,3 +104,89 @@ Call Stack (most recent call first):
 
 * Change the value to remove the trailing `\`.
 * Change the format of your `LY_3RDPARTY_PATH` to use the platform-agnostic `/` path separator.
+
+## Long build times
+
+**Issue:** Building takes longer than expected.
+
+**Remedy:** There are several steps to troubleshooting builds that are taking a long time.
+1. Use a build command that measures the amount of time taken so you can compare build times when troubleshooting.  On Windows you can use the PowerShell `measure-command` to measure build times, and on linux `time`.
+{{< tabs name="CMake build timing example" >}}
+{{% tab name="Windows" %}}
+
+```cmd
```

Comment:
For better PowerShell syntax highlighting, use `pwsh` in all your PowerShell examples.
```suggestion
```pwsh
```